src/main/scala/io/sdkman/changelogs/KarafMigrations.scala

```diff
@@ -0,0 +1,29 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "025")
+class KarafMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_karaf_candidate", author = "tonit")
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "karaf",
+      name = "Apache Karaf",
+      description = "Karaf is a polymorphic applications-runtime.",
```

Comment:
Would you please be able to provide something slightly more descriptive for this field? You can see examples of what the other projects provide when you issue the `sdk list` command.